folium/plugins/geocoder.py

```diff
@@ -19,6 +19,16 @@ class Geocoder(JSCSSMixin, MacroElement):
         Choose from 'topleft', 'topright', 'bottomleft' or 'bottomright'.
     add_marker: bool, default True
         If True, adds a marker on the found location.
+    geocodeProvider: str, default 'nominatim'
```

Comment:
please use snake_case in the Python code. The `parse_options` function turns them into camelCase.